docs/.gitignore

```diff
@@ -15,3 +15,6 @@ data/v*
 
 # Windows ADS files in docs.
 **/*:Zone.Identifier
+
+# Copy of versions json from data dir to redirect azure function
+api/fallback/versions.json
```

Comment:
I've noticed that running `pnpm clean` in the root of the repo causes some files in here to be deleted; should more files in this folder be ignored? The whole folder? Or does the clean script need to be fixed?

Reply:
I looked into the clean script, I think it would be better to fix the clean script since its deleting javascript files, should it be do that? I think maybe as a short term solution ill change the .gitignore

Reply:
I tried changing it to `api/`, `api/*`, `api/fallback/` and a couple others and they didnt stop the deleting. when I created this function it came with a gitignore, should I readd that gitignore? into the root of the /docs/api folder

Reply:
The clean thing is separate from the gitignore thing.

If the files are generated during the build, they should be ignored. The ones that are already added will need to be deleted - then future builds won't identify those generated files as changed, and that might address the ignore issue if the entire contents of that folder is build generated.

Reply:
These files are not generated, they need to be in the repo for the azure function to work. The only file that is generated is the versions.json

Reply:
Ok, you'll need to fix the clean script then. But you can do it separately.

Reply:
I noticed 4 years ago Curtis added a clean script targeting the contents of `/docs/api/`, this was not needed. The API docs are already not being committed via the .gitignore and the API docs are located in the `/docs/data/api/` folder instead. Deleted all references of deleting the contents of `/docs/api/`